src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.DebugVerifier.cs

```diff
@@ -187,6 +200,31 @@ private void VisitBinaryOperatorChildren(BoundBinaryOperatorBase node)
                 Visit(node.BoundContainingTypeOpt);
                 return null;
             }
+
+            public override BoundNode? VisitSwitchExpressionArm(BoundSwitchExpressionArm node)
```

Comment:
>VisitSwitchExpressionArm [](start = 39, length = 24)

What is the actual change in behavior for a switch expression here? What does base implementation do differently and why is that a problem?

Reply:
The base unconditionally visits the when clause. This is an issue for the reason in the comment.

---
In reply to: [441175291](https://github.com/dotnet/roslyn/pull/42403#discussion_r441175291) [](ancestors = 441175291)